core/templates/pages/collection-editor-page/services/collection-node-creator-backend-api.service.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service to create new collectio node
+ */
+// eslint-disable-next-line max-len
+require('pages/collection-editor-page/services/collection-node-creator.constants.ajs.ts');
+
+require('domain/utilities/url-interpolation.service.ts');
+
+angular.module('oppia').factory('CollectionNodeCreatorBackendApiService', [
+  '$http', 'UrlInterpolationService', 'COLLECTION_NODE_CREATOR_DATA_URL',
+  function($http, UrlInterpolationService, COLLECTION_NODE_CREATOR_DATA_URL) {
```

Comment:
Can you write this file in Angular?

Reply:
Explaining latest change via comment:
In latest changes:--
I have pre-existing service named `exploration-creation.service` and create new function in that and used that function in `collection-node-creator.directive` to make a post request at `exploration/create_new`